### PR TITLE
#175335976 update email content

### PR DESCRIPTION
**Pull request by tobieeniafe**
#### What does this PR do?
Update all email contents

#### Description of Task to be completed?
Update all email contents

#### How should this be manually tested?
`PUT /api/v1/offer/accept`

#### Any background context you want to provide?
[Email templates](https://docs.google.com/presentation/d/1l9XZHtV9-uFomT5bjZF5AAcE7pcOq8WrHmFUvHOdSjc/edit?ts=5f55d6f2#slide=id.p)

#### What are the relevant pivotal tracker stories?
#175335976

### CODE REVIEW

File: server/controllers/offer.controllers.js [hunk 1]
```diff
@@ -39,8 +39,9 @@ const OfferController = {
       .then((offer) => {
         const offerResponse = offer[0];
         const vendor = offerResponse.vendorInfo;
+        vendor.firstName = `${vendor.firstName}'s team`;
         const user = offerResponse.userInfo;
-        const contentTop = `Your offer on ${offerResponse.propertyInfo.name} has been accepted. Check your dashboard for more details.`;
+        const contentTop = `Note that your offer to ${user.lastName} ${user.firstName} on ${offerResponse.propertyInfo.name} has been accepted. Check your dashboard for more details.`;
```
Comment (andela-hpopoola) [inline server/controllers/offer.controllers.js:44]: I think the name here should be gotten from Enquiry, not User. Remember, the user might be buying the property for another person.

**Review by andela-hpopoola — CHANGES_REQUESTED** [review]
Thank you for working on this. I left a comment on ensuring that the right name is used.

**Review by andela-hpopoola — APPROVED** [review]
Thank you for working on this. Looks good to me.

**Comment by tobieeniafe** [comment]
> Thank you for working on this. Looks good to me. I will approve when you fix the failing test.

I don't understand what's happening, but the test pass locally, but keep failing on travis here also

**Review by andela-hpopoola — APPROVED** [review]
Thank you for updating this. Feel free to merge this when ready.

**Pull request merged**